app/controllers/api/status_controller.rb

```diff
@@ -5,6 +5,12 @@ class Api::StatusController < Api::BulkProjectController
 
   def check
     serializer = OptimizedProjectSerializer.new(projects, project_names, internal_key: internal_api_key?)
-    render json: Oj.dump(serializer.serialize, use_as_json: true, mode: :rails)
```

Comment:
I removed use_as_json because the Oj docs say it is ignored when combined with `mode: :rails`